scss/tech.scss

```diff
@@ -0,0 +1,39 @@
+.tech-border {
+  border: 1px solid $green;
+  border-radius: 5px;
+  height: 350px;
+  width: 200px;
+
+  .tech-container {
+    align-items: center;
+    background-color: $green;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: flex-start;
+    left: 10px;
+    position: relative;
+    top: 10px;
+    width: 100%;
+
+    .tech-title,
+    .tech-description,
+    .tech-image {
+      margin: 10px 0;
+    }
+
+    .tech-description {
+      text-align: center;
+      margin: 10px;
+    }
+
+    .tech-image {
+      @include image-size(100px);
+    }
+
+    .tech-title {
+      font-weight: 700;
```

Comment:
Use `font-weight: bold;` here.  `700` has not a clear meaning. `bold` instead is more generic and clear to all developers.